crates/rpc/rpc-types/src/eth/transaction/signature.rs

```diff
@@ -9,13 +9,18 @@ pub struct Signature {
     pub r: U256,
     /// The S field of the signature; the point on the curve.
     pub s: U256,
+    // TODO: change these fields to an untagged enum for `v` XOR `y_parity` if/when CLs support it.
+    // See <https://github.com/ethereum/go-ethereum/issues/27727> for more information
     /// For EIP-155, EIP-2930 and Blob transactions this is set to the parity (0 for even, 1 for
     /// odd) of the y-value of the secp256k1 signature.
     ///
     /// For legacy transactions, this is the recovery id
     ///
     /// See also <https://ethereum.github.io/execution-apis/api-documentation/> and <https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_gettransactionbyhash>
     pub v: U256,
+    /// The y parity of the signature. This is only used for typed (non-legacy) transactions.
```

Comment:
TIL

Reply:
_super pedantic_ correction on the comment: each EIP-2718 tax type defines its own schema, so it's really only correct to say that yParity is used for tx types 1 and 2 (and probably blob txs but I haven't looked yet). In theory a new tx type could be defined that uses `v` again. 

Not worth changing the comment for IMO but I just couldn't resist the "um, actually" urge. :)